c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -43,6 +43,7 @@ vars:
     mapfile_data_subselect: 'SELECT ST_Collect(ra.area) FROM main.restrictionarea AS ra, main.role_restrictionarea AS rra, main.layer_restrictionarea AS lra, main.treeitem AS la WHERE rra.role_id = %role_id% AND rra.restrictionarea_id = ra.id AND lra.restrictionarea_id = ra.id AND lra.layer_id = la.id AND la.name = '
     # mapserver filter on role id only
     mapfile_data_noarea_subselect: 'SELECT rra.role_id FROM {mapserver_join_tables} WHERE rra.restrictionarea_id = ra.id AND lra.restrictionarea_id = ra.id AND lra.layer_id = la.id AND la.name = '
+    geoserver: false
```

Comment:
Any reason for having two `geoserver` variable in `CONS_vars.yaml`? Isn't the one on `mapservproxy` sufficient?